### DISCUSSION TITLE
Motor kilitleri

### DISCUSSION THREAD

**Discussion by gokhanselen** [Jyers/Marlin#414] [Ideas]
Arkadaşlar Yazdırma sırasında baskı bozulduğunda veya bir nedenle stop yaptığımızda tabla hareket etmiyor bunu nasıl çözeriz kendisi tabla kilidini açsın
Birde Ekranda katman yüksekliğini göremez miyiz mesela dosya ismi yazan yerde

**Comment by Jyers** [comment]
Both problems can be fixed with octoprint. You can add some gcode to the "after cancel" script to move the head, and you can install a plugin to display the layer height.

**Reply by gokhanselen** [reply]
Thank you for your attention and response, but I have no knowledge of editing fw. Can I find the gcodes ready
Especially in my country, there are many people who use your software without their knowledge, and those who do not share information.
I am using google translate. I am already sorry for the mistakes.
Stay happy, thank you for your ingenuity in answering

**Reply by VashBaldeus** [reply]
> 
> 
> Thank you for your attention and response, but I have no knowledge of editing fw. Can I find the gcodes ready
> Especially in my country, there are many people who use your software without their knowledge, and those who do not share information.
> I am using google translate. I am already sorry for the mistakes.
> Stay happy, thank you for your ingenuity in answering

He means that in the Slicing software (Cura, PrusaSlicer) you can put commands in the End or After Cancel area a few commands to do what you want. There's a huge wiki about Marlin commands, what this will do is add those commands to each STL file you slice and save in gcode.

**Reply by gokhanselen** [reply]
Where can I find information about Marlin commands?

**Reply by Jyers** [reply]
https://marlinfw.org/meta/gcode/
